site/src/examples/slider/WithInput.tsx

```diff
@@ -44,22 +41,17 @@ export const SingleWithInput = () => {
   const handleInputChange = (event: ChangeEvent<HTMLInputElement>) => {
     const inputValue = event.target.value;
     setInputValue(inputValue);
+    setValue(Number.parseFloat(inputValue));
   };
 
-  const handleChange: SliderChangeHandler = (value: SliderValue) => {
+  const handleChange = (_e: SyntheticEvent, value: number) => {
     setValue(value);
   };
 
-  useEffect(() => {
-    const valid = validateSingle(inputValue, bounds);
```

Comment:
Looks like we removed validation from the single slider ?
Range still has it but not single